test/helpers-asLink.test.ts

```diff
@@ -110,7 +110,7 @@ it("resolves a link to web field", () => {
 		url: "https://prismic.io",
 	};
 
-	expect(asLink(field, linkResolver), "https://prismic.io");
+	expect(asLink(field, linkResolver)).toBe("https://prismic.io");
```

Comment:
Weird those tests were passing 🤔

Reply:
Yeah, it would be nice if Vitest was smart enough to throw an error if an assertion isn't called on `expect()`. In this case, the tests passed because `expect()`'s second argument is a message, and `"https://prismic.io"` is a valid message.